test/Dialect/FIRRTL/grand-central.mlir

```diff
@@ -76,6 +85,11 @@ firrtl.circuit "InterfaceGroundType" attributes {
 // CHECK-SAME:   @InterfaceGroundType
 // CHECK-SAME:   #hw.innerNameRef<@InterfaceGroundType::@dut>
 // CHECK-SAME:   #hw.innerNameRef<@DUT::@b>
+// CHECK-NEXT: sv.verbatim "assign {{[{][{]0[}][}]}}.baz = {{[{][{]1[}][}]}}.{{[{][{]2[}][}]}}.{{[{][{]3[}][}]}}[1].d;"
+// CHECK-SAME:   #hw.innerNameRef<@DUT::@__View_Foo__>
+// CHECK-SAME:   @InterfaceGroundType
+// CHECK-SAME:   #hw.innerNameRef<@InterfaceGroundType::@dut>
+// CHECK-SAME:   #hw.innerNameRef<@DUT::@c>]
```

Comment:
Eventually, we'll need to make this: `{{0}}.baz = {{1}}.{{2}}[{{3}}].{{4}}` or something like that.  This will require support for an inner name ref that tracks a field ID.  This is probably just a generalization of the field ID as opposed to a new field ref type.

Reply:
An inner name ref that tracks a field ID makes a lot sense!